geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/stats/StatsCountCommand.scala

```diff
@@ -32,12 +32,18 @@ trait StatsCountCommand[DS <: DataStore with HasGeoMesaStats] extends DataStoreC
 
     if (params.exact) {
       Command.user.info("Running stat query...")
+      val count = ds.stats.getCount(sft, filter, params.exact).map(_.toString).getOrElse("Unknown")
+      Command.output.info(s"Count: $count")
+    } else {
+      ds.stats.getCount(sft, filter, params.exact).map(_.toString) match {
+        case None =>
+          Command.output.info("Estimated count: Unknown")
+          Command.output.info("Re-run with --no-cache to get an exact count")
+
+        case Some(count) =>
+          Command.output.info(s"Estimated count: $count")
```

Comment:
I assume this is just clean up from other discussions?

Reply:
yeah, you got me ;)